### PR TITLE
[TW-1220] Trigger token scan

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Updated screenshots and steps for creating a public workspace and changing visibility to public, and added a bullet to the use cases that trigger the Secret Scanner.

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 1]
```diff
@@ -88,7 +91,7 @@ To convert an existing workspace to a public workspace, do the following:
 
 > You must be a workspace [Admin](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) to change the visibility of a workspace.
 <!--  -->
-> If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), you need a approval from a user with the [Community Manager role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to change a workspace's visibility. In the **Overview** tab, select **Workspace Settings**. Under **Visibility**, select **Public** from the dropdown list, then **Request to Change Visibility**.
+> If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), you need approval from a user with the [Community Manager role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to change a workspace's visibility. In the **Overview** tab, select **Workspace Settings**. Under **Visibility**, select **Public** from the dropdown list, then **Request to Change Visibility**.
```
Comment (nerminamiller-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:94]:
```suggestion
> If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), changing workspace visibility requires a [Community Manager](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) approval. To request this change, select **Workspace Settings**, then under **Visibility**, select **Public > Request to Change Visibility**.
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 2]
```diff
@@ -69,14 +69,17 @@ To create a public workspace, do the following:
...
 1. Select **Workspaces** from the header, then select **Create Workspace**.
 
-    <img alt="Workspace menu" src="https://assets.postman.com/postman-docs/workspace-switcher-v9.19.jpg" width="300px"/>
+    <img alt="Workspace menu" src="https://assets.postman.com/postman-docs/v10/workspace-switcher-v10.jpg" width="500px"/>
 
 1. Specify a workspace **Name** and **Summary**.
     > Within a team, you can't have two public workspaces with the same name.
 1. Under **Visibility**, select **Public**.
+
+    > If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), you need approval from a user with the [Community Manager role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to create a public workspace. Add an optional note and select **Create Workspace** to send the request and create a team workspace until a Community Manager approves it. Sending the request also triggers the [Secret Scanner](/docs/administration/token-scanner/).
```
Comment (nerminamiller-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:78]:
```suggestion
    > If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), creating a public workspace requires a [Community Manager](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) approval. Add an optional note and select **Create Workspace**. A request is sent to the Community Manager. Sending the request also triggers the [Secret Scanner](/docs/administration/token-scanner/).
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 3]
```diff
@@ -69,14 +69,17 @@ To create a public workspace, do the following:
 
 1. Select **Workspaces** from the header, then select **Create Workspace**.
```
Comment (nerminamiller-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:70]:
```suggestion
1. From the top menu, select **Workspaces > Create Workspace**.
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 4]
```diff
@@ -69,14 +69,17 @@ To create a public workspace, do the following:
...
 1. Under **Visibility**, select **Public**.
+
+    > If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), you need approval from a user with the [Community Manager role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to create a public workspace. Add an optional note and select **Create Workspace** to send the request and create a team workspace until a Community Manager approves it. Sending the request also triggers the [Secret Scanner](/docs/administration/token-scanner/).
+
 1. Select **Create workspace**.
 
-    <img alt="Create public workspace" src="https://assets.postman.com/postman-docs/create-public-wkspc-v9.5.jpg" width="400px"/>
+    <img alt="Create public workspace" src="https://assets.postman.com/postman-docs/v10/create-public-wkspc-v10.jpg" width="500px"/>
 
 > You can use workspace templates to help set up your new public workspace. For more information, see [Creating workspaces with a template](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/#creating-workspaces-with-a-template).
```
Comment (nerminamiller-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:84]:
```suggestion
> You can set up your new public workspace using a template. For more information, see [Creating workspaces with a template](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/#creating-workspaces-with-a-template).
```

**Review by nerminamiller-postman — APPROVED** [review]
LGTM with some minor suggestions

File: src/pages/docs/administration/token-scanner.md [hunk 5]
```diff
@@ -42,7 +42,8 @@ The Postman Secret Scanner scans public workspaces to detect any exposed secrets
 
 A scan starts whenever team members do any of the following actions:
 
-* Change the workspace visibility to Public.
+* Make a workspace public.
+* Send a request to a [Community Manager](/docs/collaborating-in-postman/roles-and-permissions/) to make a workspace public.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/administration/token-scanner.md:46]:
```suggestion
* Send a request to a [Community Manager](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to make a workspace public.
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:78]: @petewitcher-postman I'm not sure how GitHub handles a second suggestion on the same line. But I wanted to suggest that we change "you are" to "you're".

Also, I think you're already planning to do this, but I wanted to add here to reference later: I think we should either remove the explanation about selecting "Create Workspace", or update this to "select Create or Create Workspace". The reason is that users will see one of the CTAs depending on the plan type they're using. For more details, please see https://github.com/postmanlabs/postman-docs/pull/4915

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

**Review by Varsh59 — APPROVED** [review]

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md (hunk 4 above)
Comment (petewitcher-postman) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md]: Good suggestion, thank you. This content has been deleted.

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md (hunk 1 above)
Comment (petewitcher-postman) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md]: Good suggestion, thank you. This content has been rewritten as a procedure.

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md (hunk 2 above)
Comment (petewitcher-postman) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md]: Agreed the creating/changing visibility procedures were incongruent so I just added a ## header to the changing visibility procedure. I think that addresses the issue, do you?

Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md]: Looks good!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**